include/oneapi/dpl/type_traits

```diff
@@ -107,11 +106,13 @@ using ::std::remove_extent;
 using ::std::remove_pointer;
 using ::std::remove_reference;
 using ::std::remove_volatile;
+using ::std::true_type;
+using ::std::underlying_type;
 #if __cplusplus <= 201703L
+using ::std::is_literal_type;
```

Comment:
Why is there a limitation on standard c++17? `is_literal_type` [removed only in C++20](https://en.cppreference.com/w/cpp/types/is_literal_type)

Reply:
Injection of `is_literal_type` is within `__cplusplus <= 201703L` guard, thus it is used with c++17 and older. It won't be used with c++20 and newer.